src/Hangfire.Tags/Dashboard/TagDashboardMetrics.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Hangfire.Dashboard;
+using Hangfire.Tags.Storage;
+
+namespace Hangfire.Tags.Dashboard
+{
+    internal static class TagDashboardMetrics
+    {
+        public readonly static DashboardMetric TagsCount = new DashboardMetric("tags:count", razorPage =>
+            {
+                long count = 0;
+                using (var tagStorage = new TagsStorage(razorPage.Storage))
+                {
+                    count = tagStorage.GetTagsCount();
```

Comment:
you can get rid of the count var since GetTagsCount() defaults to 0

Reply:
good catch